#!/usr/bin/env python
PACKAGE = "carote"

from dynamic_reconfigure.parameter_generator_catkin import *

gen=ParameterGenerator()

gen.add("enabled",int_t,0,"enable/disable controller",0,0,1)
gen.add("rate",double_t,0,"control rate (in hertz)",120.0,10.0,250.0)
gen.add("Kp",double_t,0,"Position gain",0.0,0,10.0)
gen.add("Kv",double_t,0,"Velocity gain",0.0,0,10.0)
gen.add("speed_max",double_t,0,"maximum speed",0.2,0.01,0.8)
gen.add("speed_min",double_t,0,"minimum speed",0.01,0.01,0.8)

exit(gen.generate(PACKAGE, "parameters", "Follower"))
